packages/core/src/DropdownMenu/DropdownMenu.test.ts

```
@@ -1,10 +1,62 @@
 import type { VueWrapper } from '@vue/test-utils'
-import { findByRole } from '@testing-library/vue'
+import { fireEvent, findAllByRole, findByRole, render } from '@testing-library/vue'
```

ESLint

[error] Expected "findAllByRole" to come before "fireEvent".

(perfectionist/sort-named-imports)